syntax = "proto3";

package shaggy;

message ChannelLevels {
  optional int32 frame_number = 1;
  optional int32 num_channels_0 = 2;
  optional bytes levels = 3;
}
